third_party/txt/src/minikin/FontCollection.cpp

```diff
@@ -45,6 +45,9 @@ const uint32_t TEXT_STYLE_VS = 0xFE0E;
 
 uint32_t FontCollection::sNextId = 0;
 
+const uint32_t kMinFallbackScore = 0x20000000;
```

Comment:
Replace the `calcFamilyScore >= kMinFallbackScore` checks with a call to `calcCoverageScore`.

`calcFamilyScore` is a combination of `calcCoverageScore` and some other scores.  If `calcCoverageScore` returns a nonzero result, then that is equivalent to `calcFamilyScore` returning a score >= 0x20000000.